python/ui/method/dropdown_page.py

```diff
@@ -0,0 +1,21 @@
+from python.common.baseui import BasicFunctions
+from selenium.webdriver.common.by import By
+
+
+class Dropdown(BasicFunctions):
+    dropdown_header_text = dropdown_list_header_text = (By.XPATH, '//*[@id="content"]/div/h3')
+    dropdown_btn = (By.XPATH, '//*[@id="dropdown"]')
+    dropdown_select_option_one = (By.XPATH, '//*[@id="dropdown"]/option[2]')
+    dropdown_select_option_two = (By.XPATH, '//*[@id="dropdown"]/option[3]')
```

Comment:
I would recommend against using individual variables for selecting options within a drop down. I would recommend using a single variable selector for the dropdown itself, then leveraging selenium's select by module to select an item by its option value within the drop down. (You would likely need a method that takes in an argument for this)